internal/fileutils/fileutils_test.go

```diff
@@ -626,3 +627,20 @@ func testCaseSensitivePath(t *testing.T, dirName, variant string) {
 		t.Fatalf("Found should match dir \nwant: %s \ngot: %s", dir, found)
 	}
 }
+
+func TestPathsMatch(t *testing.T) {
+	if runtime.GOOS != "darwin" {
+		t.Skip("PathsMatch is only tested on macOS")
+	}
+	p1 := "/tmp"
+	p2 := "/private/tmp"
+	v, err := PathsMatch(p1, p2)
+	require.NoError(t, err, errs.JoinMessage(err))
+
+	v1, err := ResolvePath(p1)
+	require.NoError(t, err, errs.JoinMessage(err))
+	v2, err := ResolvePath(p2)
+	require.NoError(t, err, errs.JoinMessage(err))
+
+	require.True(t, v, "PathsMatch should return true, path1: %s, path2: %s", v1, v2)
+}
```

Comment:
nit: This test should run on all platforms and be expanded a bit. Right now it's covering a very narrow use-case for what could become a widely used function.

Reply:
I've only experienced this issue on Mac so that's what I targeted. Can definitely expand if we find this issue on other platforms.